packages/lesswrong/components/recommendations/RecommendationsList.jsx

```diff
@@ -11,7 +11,7 @@ class RecommendationsList extends Component {
       return <PostsLoading/>
     
     const improvedRecommendationsTooltip = <div>
-      LessWrong keeps track of what recommended posts logged-in users have read. Login to get recommended posts you haven't read before.
+      {getSetting('forumType', 'LessWrong') !== 'LessWrong' && 'The '}{getSetting('title')} keeps track of what recommended posts logged-in users have read. Login to get recommended posts you haven't read before.
```

Comment:
```suggestion
      {(getSetting('forumType', 'LessWrong') !== 'LessWrong') && 'The '}{getSetting('title')} keeps track of what recommended posts logged-in users have read. Login to get recommended posts you haven't read before.
```

Reply:
I'm not sure this is great practice. Works for now I guess but there could easily be other forums that don't need a 'The' someday.

Reply:
I have some ideas of how to change this right. Agree this is hacky, agree with your resignation that it works for now.

Reply:
Is there a reason for the parens? The `&&` gets evaluated after the `!==`
https://www.sitepoint.com/javascript-operators-conditionals-functions/

Reply:
I'm not 100% sure what the rules our lint yells at us for some instances of "having multiple operations next to each other without separating them out with parens" and I was erring on the safe side. But I guess if it's passing lint it's probably fine.